schemas/pagination-result.schema.yaml

```diff
@@ -4,7 +4,9 @@ title: PaginationResult
 type: object
 properties:
   results:
-    type: object
+    type:
+      - object
```

Comment:
I'm wondering why it was `object` in the first place. If it's a paginated result, shouldn't it always be an array?

Reply:
This is json-schema weirdness. The desired result is to have something akin to a generic `PaginatedResult<T>` where `results` is of type `T[]`. But json-schema doesn't really support generics. (unless you count this... https://json-schema.org/blog/posts/dynamicref-and-generics)

People will be "extending" this schema through `allOf` and replacing results with their own type. If anything we should probably just drop `results` from this type. If i'm remembering correctly, it was including because it just _looks_ better.

Reply:
setting it to `true` might be another good option. Which is basically just an `any` type, since it will be constrained by any extender.

```
properties:
  results: true
```

Reply:
Hmmm... I think dropping it seems more reasonable, is there documentation about the setting to "true" option, that seems like a hack from a code perspective but maybe that's reasonable for a spec